controllers/webhook_cabundler.go

```diff
@@ -12,6 +18,64 @@ type k8s struct {
 	clientset kubernetes.Interface
 }
 
+//InitWebhookCertificates creates and injects req certs by the k8s webhooks
+func InitWebhookCertificates(certDir string) error {
+
+	secretName := os.Getenv("WEBHOOK_SECRET_NAME")
+	namespace := os.Getenv("SERVICE_NAMESPACE")
+	validatingWebhookConfigurationName := os.Getenv("VALIDATING_WEBHOOK_CONFIGURATION")
+	mutatingWebhookConfigurationName := os.Getenv("MUTATING_WEBHOOK_CONFIGURATION")
+	val := os.Getenv("CERTIFICATE_SANS")
+	dnsNames := strings.Split(val, ",")
+
+	if len(secretName) == 0 || len(namespace) == 0 || len(validatingWebhookConfigurationName) == 0 ||
+		len(mutatingWebhookConfigurationName) == 0 || len(dnsNames) == 0 {
+		os.Exit(1)
+		return errors.New("Need ENVS: WEBHOOK_SECRET_NAME, SERVICE_NAMESPACE, " +
+			"VALIDATING_WEBHOOK_CONFIGURATION, MUTATING_WEBHOOK_CONFIGURATION, CERTIFICATE_SANS")
+	}
```

Comment:
I took me a while to realize you can define flags in a package like this one, and they'll still show up when running the main program.

Reply:
I think that's worth thinking about. However, I think we would want to do that as a separate PR since this one is just meant to move things around/clean up code. Can you create a ticket for this?

Reply:
done https://github.com/ForgeRock/secret-agent/issues/27